pylabrobot/resources/celltreat/plates.py

```diff
@@ -0,0 +1,100 @@
+from typing import Optional
+from pylabrobot.resources.height_functions import _compute_height_from_volume_cylinder, _compute_volume_from_height_cylinder
+
+from pylabrobot.resources.plate import Lid, Plate
+from pylabrobot.resources.utils import create_equally_spaced_2d
+from pylabrobot.resources.well import Well, WellBottomType
+
+
+def CellTreat_96_WP_U(name: str, lid: Optional[Lid] = None) -> Plate:
+    """
+    CellTreat cat. no.: 229590
+    - Material: Polystyrene
+    - Tissue culture treated: No
+    """
+    WELL_UBOTTOM_HEIGHT = 2.81 # absolute height of cylindrical segment, measured
+    WELL_DIAMETER = 6.69 # measured
+
+    return Plate(
+        name=name,
+        size_x=127.76,
+        size_y=85.11,
+        size_z=14.30,  # without lid
+        lid=lid,
+        model=CellTreat_96_WP_U.__name__,
+        items=create_equally_spaced_2d(
+            Well,
+            num_items_x=12,
+            num_items_y=8,
+            dx=11.05,  # measured
+            dy=7.75,  # measured
+            dz=1.92,  # calibrated manually
+            item_dx=8.99,
+            item_dy=8.99,
+            size_x=6.35,
+            size_y=6.35,
+            size_z=10.04,
+            bottom_type=WellBottomType.U,
+        ),
+    )
+
+
+def CellTreat_96_WP_U_Lid(name: str) -> Lid:
+    """
+    CellTreat cat. no.: 229590
+    - Material: Polystyrene
+    - Tissue culture treated: No
+    """
+    return Lid(
+        name=name,
+        size_x=127.762,
+        size_y=85.471,
+        size_z=10.71,  # measured
+        nesting_z_height=8.30,  # measured as height of plate "plateau"
+        model=CellTreat_96_WP_U_Lid.__name__,
+    )
+
+
+def CellTreat_6_WP_Flat(name: str, lid: Optional[Lid] = None) -> Plate:
+    WELL_RADIUS = 17.5
+
+    return Plate(
+        name=name,
+        size_x=127.0,
+        size_y=86.0,
```

Comment:
have you verified these are perfect? in https://github.com/PyLabRobot/pylabrobot/pull/180 we discovered the plate dimensions were wrong in VENUS (127.0x86.0) and I have spread this issue by copying it everywhere.

the difference is tiny, but it's good to have high standards for labware moving forward because we have the opportunity to build the best library in the world.

the official guide, which might be wrong, says 127.8 by 85.38 (https://www.celltreat.com/wp-content/uploads/6-Well-Plate.pdf)

we should obviously deviate from docs if reality measurements are different, but numbers have to be based on either docs or measurements, and not simply copy paste :)

Reply:
great find, I'll change this.